### ISSUE TITLE
[Improvements] Changing dates

### ISSUE THREAD

**Issue by Chouchen** [nickv-nextcloud/gadgetbridge#2]
Hi,

I just installed it and it worked great with my miband2.

It's just weird not to be able to change date or navigate between dates like in the android app.

It should be possible easily with `_loadDevice: function(date)` available with the date formated like `YYYY/MM/DD/HH/mm` this.

Thanks for this !

**Comment by nickvergessen** [comment]
Yeah, I only prototyped this first version back then, waiting for the auto-export and the final data format of that.

Since it seems like that will just be the export file as well, I will now continue to work on this, although I have to admit I'm not very good with UI stuff.
